Makefile

```diff
@@ -99,7 +99,7 @@ ifeq (, $(shell which controller-gen))
 	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$CONTROLLER_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.2 ;\
+	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0 ;\
```

Comment:
Do we need to update this?

Reply:
Version 07.0:
Bump to kubernetes 1.22.1, remove support for legacy v1beta1 CRDs and webhooks

https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.7.0